mobile/src/main/res/values/strings.xml

```diff
@@ -5,17 +5,34 @@
 
     <!-- General app strings -->
     <string name="openhab_service_type" translatable="false">_openhab-server._tcp.local.</string>
+    <string name="main_menu">Main menu</string>
     <string name="app_preferences_name">Settings</string>
     <string name="app_notifications">Notifications</string>
     <string name="dlg_notification_title">Notification message</string>
+    <string name="favorites">Favorites</string>
```

Comment:
@Tony-Hu Looks like something got wrong with your merge. This and many other strings have been removed.

Reply:
@mueller-ma I assumed he wasn't done yet. There's also a lot of code this PR reintroduces that has been refactored out since the last PR was created.

Reply:
Correct. I kept everything there as the merge result. And I'm currently deleting the unnecessary part to let it fit with the current version of openHAB.